src/svgSprite.ts

```diff
@@ -504,5 +502,17 @@ export const svgSprite = (): string => `
+  </symbol>
+
+  <symbol viewBox="0 0 21 21" id="svg-retain-data">
```

Comment:
[the same ](https://github.com/ChildMindInstitute/mindlogger-admin/pull/77/files#r1054224428)